src/elements/content-sidebar/versions/StaticVersionsItem.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @flow
+ * @file Static Versions Item component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import messages from './messages';
+import sizeUtil from '../../../utils/size';
+import VersionsItemBadge from './VersionsItemBadge';
+import { ReadableTime } from '../../../components/time';
+import PlainButton from '../../../components/plain-button';
+
+import './VersionsItem.scss';
+
+type Props = {
+    versionNumber: string,
+    versionTime: string,
+};
+
+const FIVE_MINUTES_MS = 5 * 60 * 1000;
+
+const StaticVersionsItem = ({ versionNumber, versionTime }: Props) => {
+    // Version info helpers
+    const versionTimestamp = new Date(versionTime).getTime();
+
+    const versionDisplayName = <FormattedMessage {...messages.versionUserUnknown} />;
+
+    const buttonClassName = classNames('bcs-VersionsItemButton', {
+        'bcs-is-disabled': false,
+        'bcs-is-selected': false,
+    });
+
+    return (
+        <div className="bcs-VersionsItem">
```

Comment:
`[nit]` can we make this wrapping element's `className` to be `bcs-StaticVersionsItem` to match the component or are we depending on this versions item class?

Reply:
Currently we are depending on this versions item class since the css is the same across the static vs actual. I didnt love that I was doing this, but at the same time it does eliminate copying of code. So I will probably work around to see what the minimum styling is and move it